### PR TITLE
Unify the user and operator grafanas

### LINKED ISSUE #4800: Open Prometheus ingress or Grafana Explore feature to shoot owners
**How to categorize this issue?**

/area monitoring
/kind enhancement

**What would you like to be added**:

Let's do one (or both) of the following:
- allow shoot owners to access the prometheus ingress of their shoot clusters
- enable the `Explore` feature in the user-facing Grafana instance (ref https://github.com/gardener/gardener/pull/4140)

**Why is this needed**:

Endusers want to analyze the performance data of their clusters more freely.
Currently they are restricted to using the provided dashboards in the user-facing Grafana instance. They are currently not able to use arbitrary queries against prometheus (and loki).
Enabling endusers to debug and analyze their clusters in more detail, will relieve operators from helping them out because of current monitoring restrictions on their side.

### PR DESCRIPTION
**Pull request by wyb1**
**How to categorize this PR?**

/area monitoring
/kind enhancement

**What this PR does / why we need it**:

Gardener provides 2 Grafanas. One of these is for end users and the other for operators. The end user Grafana only has a subset of the functions that the operator Grafana has. The purpose of this change is to unify the operator and user Grafanas into one. This will improve collaboration between operators and users and simplify deployment. 

**Which issue(s) this PR fixes**:
Fixes #4800

**Special notes for your reviewer**:

* Observability secrets are no longer automatically updated. They are treated as user observability credentials
* The "deprecated" grafanas resources (i.e. resources with the labels `component=grafana,role=users|operators`) will always be deleted at the beginning of the `DeploySeedGrafana` function. This migration logic should be deleted in the future.
* The ingress prefix used for this new grafana will be `gu` the same as the user's prefix before.

**Release note**:

```other operator|user
User and operator Grafanas are now unified. This means that there is only one Grafana deployment and users and operators have the same Grafana functionalities.
```

### CODE REVIEW

File: pkg/operation/botanist/monitoring.go [hunk 1]
```diff
@@ -502,13 +502,27 @@ func (b *Botanist) DeploySeedGrafana(ctx context.Context) error {
...
 		gutil.ShootProjectSecretSuffixMonitoring,
 		map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleMonitoring},
 		map[string]string{"url": "https://" + b.ComputeGrafanaUsersHost()},
 		credentialsUsersSecret.Data,
-	)
+	); err != nil {
+		return err
+	}
+
+	if err := b.syncShootCredentialToGarden(
```
Comment (plkokanov) [inline pkg/operation/botanist/monitoring.go]:
```suggestion
	return b.syncShootCredentialToGarden(
```

File: pkg/utils/gardener/shoot.go [hunk 2]
```diff
@@ -175,6 +175,8 @@ const (
 	ShootProjectSecretSuffixOldSSHKeypair = v1beta1constants.SecretNameSSHKeyPair + ".old"
 	// ShootProjectSecretSuffixMonitoring is a constant for a shoot project secret with suffix 'monitoring'.
 	ShootProjectSecretSuffixMonitoring = "monitoring"
+	// ShootProjectSecretSuffixMonitoring is a constant for a shoot project secret with suffix 'monitoring-operator'.
```
Comment (plkokanov) [inline pkg/utils/gardener/shoot.go]:
```suggestion
	// ShootProjectSecretSuffixMonitoringOperator is a constant for a shoot project secret with suffix 'monitoring-operator'.

File: pkg/utils/gardener/shoot.go [hunk 3]
```diff
@@ -185,6 +187,7 @@ func GetShootProjectSecretSuffixes() []string {
 		ShootProjectSecretSuffixSSHKeypair,
 		ShootProjectSecretSuffixOldSSHKeypair,
 		ShootProjectSecretSuffixMonitoring,
+		ShootProjectSecretSuffixMonitoringOperator,
```
Comment (plkokanov) [inline pkg/utils/gardener/shoot.go]: https://github.com/gardener/gardener/blob/24dc519d70c664bb64ebe46808320ae9595d3cfc/pkg/utils/gardener/shoot_test.go#L286 
Should be updated

**Review by plkokanov — CHANGES_REQUESTED** [review]
This will make the operator credentials accessible by users. Is this expected/planned?

**Review by ialidzhikov — COMMENTED** [review]
Additionally, such change would make `grafana-users` completely obsolete as it is a subset of `grafana-owners`. What are the future plans - do you plan to drop completely `grafana-users`?

**Comment by rfranzke** [comment]
> Additionally, such change would make `grafana-users` completely obsolete as it is a subset of `grafana-owners`. What are the future plans - do you plan to drop completely `grafana-users`?

IIUC, this is the plan, yes. Only one Grafana instance with all the information.

@wyb1 Instead of syncing this secret now, should we just merge the "operators" Grafana into the "users" Grafana and drop the "operators" Grafana? Otherwise, we are now opening "another API" which we have to close soon again.

**Comment by wyb1** [comment]
>Instead of syncing this secret now, should we just merge the "operators" Grafana into the "users" Grafana and drop the "operators" Grafana? Otherwise, we are now opening "another API" which we have to close soon again.

This sounds like a better idea. I will update the PR to do this instead.

**Comment by wyb1** [comment]
Sorry for the long wait, the change was much more involved than I initially thought. I am still verifying that everything works as expected, but I feel that I can share the current state. The PR can stay in a draft/WIP state for now.

**Comment by rfranzke** [comment]
FYI @grolu @holgerkoser @petersutter

File: pkg/operation/botanist/monitoring.go [hunk 4]
```diff
@@ -49,8 +49,7 @@ import (
 )
 
 const (
-	secretNameIngressOperators = v1beta1constants.SecretNameObservabilityIngress
-	secretNameIngressUsers     = v1beta1constants.SecretNameObservabilityIngressUsers
+	secretNameIngressUsers = v1beta1constants.SecretNameObservabilityIngressUsers
```
Comment (rfranzke) [inline pkg/operation/botanist/monitoring.go:49]: Just call it `secretNameIngress`?

File: pkg/operation/botanist/monitoring.go [hunk 5]
```diff
@@ -628,8 +619,12 @@ func (b *Botanist) deployGrafanaCharts(ctx context.Context, credentialsSecret *c
 	)
 }
 
-// DeleteGrafana will delete all grafana instances from the seed cluster.
 func (b *Botanist) DeleteGrafana(ctx context.Context) error {
```
Comment (rfranzke) [inline pkg/operation/botanist/monitoring.go:621]: Doc string missing

File: pkg/operation/operation.go [hunk 6]
```diff
@@ -604,7 +604,7 @@ func (o *Operation) WantsGrafana() bool {
 	return o.Shoot.Purpose != gardencorev1beta1.ShootPurposeTesting && (helper.IsMonitoringEnabled(o.Config) || helper.IsLokiEnabled(o.Config))
 }
 
-// ComputeGrafanaUsersHost computes the host for operators Grafana.
+// ComputeGrafanaUsersHost computes the host for users Grafana.
```
Comment (rfranzke) [inline pkg/operation/operation.go]: Rename all of this to `ComputeGrafanaHost` only?

File: pkg/operation/botanist/secrets.go [hunk 7]
```diff
@@ -91,10 +91,6 @@ func (b *Botanist) lastSecretRotationStartTimes() map[string]time.Time {
 			rotation[v1beta1constants.SecretNameSSHKeyPair] = shootStatus.Credentials.Rotation.SSHKeypair.LastInitiationTime.Time
 		}
 
-		if shootStatus.Credentials.Rotation.Observability != nil && shootStatus.Credentials.Rotation.Observability.LastInitiationTime != nil {
-			rotation[v1beta1constants.SecretNameObservabilityIngressUsers] = shootStatus.Credentials.Rotation.Observability.LastInitiationTime.Time
-		}
-
```
Comment (rfranzke) [inline pkg/operation/botanist/secrets.go]: Why did you remove this? We still want the credentials to be rotated when the `gardener.cloud/operation=rotate-observability-credentials` is annotated, don't we?

Comment (rfranzke) [inline pkg/operation/botanist/secrets.go]: Why did you remove this? We still want the credentials to be rotated when the `gardener.cloud/operation=rotate-observability-credentials` is annotated, don't we?

File: docs/usage/logging.md [hunk 8]
```diff
@@ -17,25 +17,21 @@ The logging stack is extended to scrape logs from the systemd services of each s
...
-
-  2. In addition to the dashboards in the User Grafana, the Operator Grafana contains several other dashboards that aim to facilitate the work of operators.
-  The operator Grafana URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard alongside with the credentials, when opened as Gardener operator.
-  Also, it can be found in the `control-plane` namespace under the `grafana-operators` ingress in the seed.
-  Operators have access to the `Explore` tab.
-  The secret with the credentials can be found in the `control-plane` (shoot--project--shoot-name) namespace under `observability-ingress-<hash>-<hash>` secrets in the seed.
-  From `Explore` tab, operators have unlimited abilities to extract and manipulate logs.
-  The Grafana itself helps them with suggestions and auto-completion.
+
+The logs are accesible via Grafana. To access them:
```
Comment (rfranzke) [inline docs/usage/logging.md]:
```suggestion
The logs are accessible via Grafana. To access them:
```

File: docs/usage/logging.md [hunk 9]
```diff
@@ -17,25 +17,21 @@ The logging stack is extended to scrape logs from the systemd services of each s
...
+
+  1. Authenticate via basic auth to gain access to Grafana.
+  The Grafana URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard alongside with the credentials.
+  Alternatively, the secret with the credentials can be found in the `control-plane` (shoot--project--shoot-name) namespace under `observability-ingress-<hash>-<hash>` secret in the seed.
```
Comment (rfranzke) [inline docs/usage/logging.md]: The secret should also still be in the project namespace in the garden cluster?

**Review by rfranzke — CHANGES_REQUESTED** [review]

File: docs/usage/logging.md [hunk 10]
```diff
@@ -17,25 +17,23 @@ The logging stack is extended to scrape logs from the systemd services of each s
...
-  Operators have access to the `Explore` tab.
-  The secret with the credentials can be found in the `control-plane` (shoot--project--shoot-name) namespace under `observability-ingress-<hash>-<hash>` secrets in the seed.
-  From `Explore` tab, operators have unlimited abilities to extract and manipulate logs.
-  The Grafana itself helps them with suggestions and auto-completion.
+
+The logs are accessible via Grafana. To access them:
+
+  1. Authenticate via basic auth to gain access to Grafana.
+  The Grafana URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard alongside with the credentials.
+  The secret is found in the project namespace with the patten `<shoot-name>.monitoring`.
```
Comment (istvanballok) [inline docs/usage/logging.md]:
```suggestion
  The secret is found in the project namespace with the pattern `<shoot-name>.monitoring`.
```

**Comment by wyb1** [comment]
Currently, the user credentials will be forcefully rotated to the operator credentials. This is probably not the behavior that we want, so we will look into changing this. Perhaps by keeping the secret `observability-ingress-users` in the control plane in favor of `observability-ingress`.

**Comment by wyb1** [comment]
The user credentials will no longer be forcefully rotated. PR is ready for review :)

**Comment by istvanballok** [comment]
We could check the alertmanager ingress when the alerting feature is activated for the shoot.

https://github.com/gardener/gardener/blob/40b0bf10522e53c55ceaf1b361a4d016a94fe741/example/90-shoot.yaml#L322-L325

-> 👍

File: pkg/operation/botanist/monitoring.go [hunk 11]
```diff
@@ -635,8 +619,13 @@ func (b *Botanist) deployGrafanaCharts(ctx context.Context, credentialsSecret *c
...
 
-// DeleteGrafana will delete all grafana instances from the seed cluster.
+// DeleteGrafana will delete all grafana resources from the seed cluster.
 func (b *Botanist) DeleteGrafana(ctx context.Context) error {
+	return common.DeleteGrafana(ctx, b.SeedClientSet, b.Shoot.SeedNamespace)
+}
+
+// DeleteDeprecatedGrafana will delete all deprecated grafana instances from the seed cluster.
+func (b *Botanist) DeleteDeprecatedGrafana(ctx context.Context) error {
 	if err := common.DeleteGrafanaByRole(ctx, b.SeedClientSet, b.Shoot.SeedNamespace, grafanaOperatorsRole); err != nil {
```
Comment (istvanballok) [inline pkg/operation/botanist/monitoring.go:628]: Maybe we could mark the functions with a TODO "to be removed" that are not needed after the migration.

File: pkg/operation/botanist/monitoring.go (hunk 4 above)
Comment (istvanballok) [reply pkg/operation/botanist/monitoring.go:49]: Eventually @wyb1 and I decided to continue using the SecretNameObservabilityIngressUsers secret name to avoid changing the secret for end users as a side effect of this PR.

**Comment by istvanballok** [comment]
As discussed with @petersutter, we would hold this change until the next Gardener release in 2 weeks (1.59), so that the Gardener dashboard changes can be prepared and released as well. cc @wyb1 

otherwise, /lgtm

/hold

**Comment by ialidzhikov** [comment]
/assign @Kristian-ZH @vlvasilev 

@Kristian-ZH @vlvasilev can you check this PR from logging point of view? IIRC you implemented a tenant isolation for `user`/`operator` in loki. Is everything that needs to be adapted from logging point of view adapted in this PR?

File: pkg/operation/common/utils.go [hunk 12]
```diff
@@ -197,6 +197,49 @@ func DeleteAlertmanager(ctx context.Context, k8sClient client.Client, namespace
...
+	}
+
+	deleteOptions := []client.DeleteAllOfOption{
+		client.InNamespace(namespace),
+		client.MatchingLabels{
+			"component": "grafana",
+		},
+	}
+
+	if err := k8sClient.Client().DeleteAllOf(ctx, &appsv1.Deployment{}, append(deleteOptions, client.PropagationPolicy(metav1.DeletePropagationForeground))...); client.IgnoreNotFound(err) != nil {
```
Comment (ialidzhikov) [inline pkg/operation/common/utils.go]: I don't think a `DeleteAllOf` (delete collection) call can return a not found error. Ref https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#delete-collection-pod-v1-core.

Do you see any other usage of `DeleteAllOf` gardener/gardener to ignore not found errors?

```suggestion
	if err := k8sClient.Client().DeleteAllOf(ctx, &appsv1.Deployment{}, append(deleteOptions, client.PropagationPolicy(metav1.DeletePropagationForeground))...); err != nil {
```

File: test/testmachinery/shoots/logging/seed_logging_stack.go [hunk 13]
```diff
@@ -161,8 +157,7 @@ var _ = ginkgo.Describe("Seed logging testing", func() {
 		)
 
 		ginkgo.By("Get Loki tenant IDs")
-		userID := getXScopeOrgID(grafanaUsersIngress.GetAnnotations())
-		operatorID := getXScopeOrgID(grafanaOperatorsIngress.GetAnnotations())
+		ID := getXScopeOrgID(grafanaIngress.GetAnnotations())
```
Comment (ialidzhikov) [inline test/testmachinery/shoots/logging/seed_logging_stack.go]:
```suggestion
		id := getXScopeOrgID(grafanaIngress.GetAnnotations())
```

File: charts/seed-monitoring/charts/grafana/templates/grafana-ingress.yaml [hunk 14]
```diff
@@ -8,16 +8,11 @@ metadata:
     nginx.ingress.kubernetes.io/auth-realm: Authentication Required
     nginx.ingress.kubernetes.io/auth-secret: {{ .Values.ingress.authSecretName }}
     nginx.ingress.kubernetes.io/auth-type: basic
-{{- if eq .Values.role "users" }}
```
Comment (ialidzhikov) [inline charts/seed-monitoring/charts/grafana/templates/grafana-ingress.yaml:11]: As we now unify to single grafana, https://github.com/gardener/gardener/blob/54bdaaac6f6dd43cb034b367459cf95acfbf4ff9/pkg/operation/botanist/component/types.go#L47-L48 no longer make sense.

Can you also sync with @Kristian-ZH and @vlvasilev about the required logging adaptations?

For example https://github.com/gardener/gardener/blob/v1.57.1/docs/extensions/logging-and-monitoring.md#how-to-expose-logs-to-the-users no longer makes sense.

File: docs/usage/logging.md [hunk 15]
```diff
@@ -17,25 +17,23 @@ The logging stack is extended to scrape logs from the systemd services of each s
 Also, in the shoot control plane an `event-logger` pod is deployed which scrapes events from the shoot `kube-system` namespace and shoot `control-plane` namespace in the seed. The `event-logger` logs the events to the standard output. Then the `fluent-bit` gets these events as container logs and sends them to the Loki in the shoot control plane (similar to how it works for any other control plane component).
 
 ### How to access the logs
-The first step is to authenticate in front of the Grafana ingress.
-There are two Grafana instances where the logs are accessible from.
-  1. The user (stakeholder/cluster-owner) Grafana consist of a predefined  `Monitoring and Logging` dashboards which help the end-user to get the most important metrics and logs out of the box. This Grafana UI is dedicated only for the end-user and does not show logs from components which could log a sensitive information. Also, the `Explore` tab is not available. Those logs are in the predefined dashboard named `Controlplane Logs Dashboard`.
```
Comment (ialidzhikov) [inline docs/usage/logging.md:53]: The `Controlplane Logs Dashboard` still exists and is broken with the changes from this PR.

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

**Comment by vlvasilev** [comment]
> /assign @Kristian-ZH @vlvasilev
> 
> @Kristian-ZH @vlvasilev can you check this PR from logging point of view? IIRC you implemented a tenant isolation for `user`/`operator` in loki. Is everything that needs to be adapted from logging point of view adapted in this PR?

The user logs are not reachable through Grafana. But the user Grafana now uses the operator identity. And because operators have access to all the logs this PR is fine. After its merge, I will drop the tenant isolation.

**Review by vlvasilev — APPROVED** [review]
/lgtm

File: pkg/operation/common/utils.go (hunk 12 above)
Comment (istvanballok) [reply pkg/operation/common/utils.go]: @wyb1 and I tested it and actually a `not found` error is returned which makes the consecutive reconciliations fail.
I'm not sure if this document lists the possible errors that can be returned by the function: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#delete-pod-v1-core

```
2022-10-21T08:42:30.013Z ERROR controller.shoot.shoot Error reconciling request
{name: local2,
namespace: garden-local,
error: Flow "Shoot cluster reconciliation" encountered task errors:
[task "Reconciling Grafana for Shoot in Seed for the logging stack" failed:
retry failed with context deadline exceeded,
last error: services "grafana-operators" not found task "Deploying Shoot monitoring stack in Seed" failed:
retry failed with context deadline exceeded,
last error: secret "observability-ingress-users" not found]}
```

File: docs/usage/logging.md (hunk 15 above)
Comment (istvanballok) [reply docs/usage/logging.md:53]: We double checked that the `Controlplane Logs Dashboard` works:
image

Note that Grafana/loki requires the `X-Scope-OrgID operator` header and that is set by the ingress, so port-forwarding to Grafana is not enough, we have to go through the ingress. If you are using the remote local setup, these are the steps:

```
kubectl port-forward -n garden svc/nginx-ingress-controller 1443:443

k get secrets -n garden-local local.monitoring -o json | jq .data.password -r | base64 -d; echo

# on the laptop
k port-forward deployments/remote-local-setup 1443
open https://gu-local--local.ingress.local.seed.local.gardener.cloud:1443/?orgId=1

# in /etc/hosts, have an entry like this
127.0.0.1 *.ingress.local.seed.local.gardener.cloud
```

File: pkg/operation/common/utils.go (hunk 12 above)
Comment (ialidzhikov) [reply pkg/operation/common/utils.go]: You should be able to remove the `client.IgnoreNotFound` from `DeleteAllOf` calls. You problem was rather that in https://github.com/gardener/gardener/pull/6749/commits/882885cc4ff9eb9914f1ce2d0f68a9c28968eb2c you removed the `client.IgnoreNotFound` from L236 which is actually not a `DeleteAllOf` call but a `Delete` call. You should preserve the  `client.IgnoreNotFound` there.

File: docs/usage/logging.md (hunk 15 above)
Comment (ialidzhikov) [reply docs/usage/logging.md:53]: I think my issue was rather that the logging was not enabled for the Seed. Sorry about this comment that is not applicable.

File: pkg/operation/common/utils.go (hunk 12 above)
Comment (ialidzhikov) [reply pkg/operation/common/utils.go]: I will also try to test on my end in a while.

Comment (istvanballok) [reply pkg/operation/common/utils.go]: Thank you for finding out the issue. I repushed the commit without the unneeded change: https://github.com/gardener/gardener/pull/6749/commits/770eaa160eee54374b4cc8761142355b8c3e3446

Comment (ialidzhikov) [reply pkg/operation/common/utils.go]: Thanks for incorporating the suggestion! The latest revision works on my end as well! 👍

File: docs/usage/logging.md [hunk 16]
```diff
@@ -17,25 +17,23 @@ The logging stack is extended to scrape logs from the systemd services of each s
...
-  The secret with the credentials can be found in the `control-plane` (shoot--project--shoot-name) namespace under `observability-ingress-<hash>-<hash>` secrets in the seed.
-  From `Explore` tab, operators have unlimited abilities to extract and manipulate logs.
-  The Grafana itself helps them with suggestions and auto-completion.
+
+The logs are accessible via Grafana. To access them:
+
+  1. Authenticate via basic auth to gain access to Grafana.
+  The Grafana URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard alongside with the credentials.
+  The secret is found in the project namespace with the pattern `<shoot-name>.monitoring`.
+  Alternatively, the secret with the credentials can be found in the `control-plane` (shoot--project--shoot-name) namespace in the `observability-ingress-users-<hash>-<hash>` secret in the seed.
```
Comment (ialidzhikov) [inline docs/usage/logging.md]: I know it is not introduced with this PR but in case you want to address a small typo:
```suggestion
  Alternatively, the secret with the credentials can be found in the `control-plane` (shoot--project--shoot-name) namespace in the `observability-ingress-users-<hash>` secret in the seed.
```

File: pkg/operation/seed/seed.go [hunk 17]
```diff
@@ -550,10 +550,6 @@ func RunReconcileSeedFlow(
 
 			filters.WriteString(fmt.Sprintln(loggingConfig.Filters))
 			parsers.WriteString(fmt.Sprintln(loggingConfig.Parsers))
-
-			if loggingConfig.UserExposed {
-				userAllowedComponents = append(userAllowedComponents, loggingConfig.PodPrefixes...)
```
Comment (ialidzhikov) [inline pkg/operation/seed/seed.go]: Maybe we should revert 083c4a45ba2a8ff70e6813f17179b966524d226d and let @vlvasilev do the complete logging cleanup in a separate PR. WDYT?

Reasoning: I assume that there are other places that have to be cleaned up/adapted:
- this `userAllowedComponents` slice now does not do much and we never append to it
https://github.com/gardener/gardener/blob/083c4a45ba2a8ff70e6813f17179b966524d226d/pkg/operation/seed/seed.go#L450
later on it is used in a fluent-bit filter https://github.com/gardener/gardener/blob/083c4a45ba2a8ff70e6813f17179b966524d226d/pkg/operation/seed/seed.go#L558

- there are filters that still contain the `user` tenant
https://github.com/gardener/gardener/blob/083c4a45ba2a8ff70e6813f17179b966524d226d/pkg/operation/seed/seed.go#L559
https://github.com/gardener/gardener/blob/083c4a45ba2a8ff70e6813f17179b966524d226d/pkg/operation/seed/seed.go#L577

File: docs/usage/logging.md [hunk 18]
```diff
@@ -17,25 +17,23 @@ The logging stack is extended to scrape logs from the systemd services of each s
 Also, in the shoot control plane an `event-logger` pod is deployed which scrapes events from the shoot `kube-system` namespace and shoot `control-plane` namespace in the seed. The `event-logger` logs the events to the standard output. Then the `fluent-bit` gets these events as container logs and sends them to the Loki in the shoot control plane (similar to how it works for any other control plane component).
 
 ### How to access the logs
-The first step is to authenticate in front of the Grafana ingress.
```
Comment (ialidzhikov) [inline docs/usage/logging.md:51]: You could also update the section in https://github.com/gardener/gardener/blob/v1.57.1/docs/extensions/logging-and-monitoring.md#grafana. Or do you prefer to leave it to the cleanup of @vlvasilev ?

File: pkg/operation/seed/seed.go (hunk 17 above)
Comment (istvanballok) [reply pkg/operation/seed/seed.go]: > Maybe we should revert https://github.com/gardener/gardener/commit/083c4a45ba2a8ff70e6813f17179b966524d226d and let @vlvasilev do the complete logging cleanup in a separate PR. WDYT?

I agree and reverted the commit as you suggested. 👍

File: docs/usage/logging.md (hunk 18 above)
Comment (istvanballok) [reply docs/usage/logging.md:51]: I think it could be done in a separate PR together with the other cleanup activities from the logging perspective.

**Comment by istvanballok** [comment]
Keeping this PR on hold for now in favor of working on another topic -> might move to the next release.

**Comment by istvanballok** [comment]
/close in favor of 
- #7007

**Comment by shafeeqes** [comment]
/close 
> in favor of
> 
> * [Unify the user and operator Grafana deployments #7007](https://github.com/gardener/gardener/pull/7007)
